text/0134-remove-span-status.md

```diff
@@ -0,0 +1,89 @@
+# Remove Span.Status
+
+**Author:** Nikita Salnikov-Tarnovski, Splunk
+
+Remove `Span.status` and related APIs and conventions.
+
+## Motivation
+
+During numerous heated discussions in various pull requests (see
+<https://github.com/open-telemetry/opentelemetry-specification/issues/599>
+<https://github.com/open-telemetry/opentelemetry-specification/pull/697>
+<https://github.com/open-telemetry/opentelemetry-specification/pull/427>
+<https://github.com/open-telemetry/oteps/pull/69>
+<https://github.com/open-telemetry/oteps/pull/86>
+<https://github.com/open-telemetry/oteps/pull/123>)
+and during Errors Working Group meetings a lot of concerns were raised about suitability of the current `Span.Status`
+field encoded as Google RPC code for recording a result of arbitrary monitored operations from various domains
+(http requests, database queries, file system operations etc). It is hard to define clear and umambigous mapping
+from those domain to a fixed enum originated from RPC.
+
+Therefore the proposal is to remove the `Span.Status` field from Trace API and OTLP.
+
+## Internal details
+
+Several areas are affected by this change and will require a coordinated effort to make this happen.
+
+### Trace API Specification
+
+* Remove all mentions of `Status` and `Set Status` API from Span section.
+* Remove whole `Status` section.
+
+### Trace SDK Specification
+
+No changes needed.
+
+### Trace SDK exporters
+
+Remove translation rules pertaining to `Span.Status`.
+
+### Span semantic conventions
+
+* From HTTP conventions, remove `Status` section.
+* In gRPC conventions, replace the requirement to set span status with the requirement to set span attribute `grpc.status`
```

Comment:
Should the OTLP exporter (optionally) do the translation of grpc and HTTP status attributes to deprecated_status for now? Or is this not worth it?

Reply:
As one of the reasons to remove `status` was the ambiguity of such translations (at least in http part), I don't think it is worth doing. And ideally we will not produce this Google codes based status anymore.

Reply:
Make sense.